Resources/Help/marked.js

```diff
@@ -1,1286 +1,2838 @@
 /**
- * marked - a markdown parser
- * Copyright (c) 2011-2014, Christopher Jeffrey. (MIT Licensed)
- * https://github.com/chjj/marked
+ * marked v5.0.1 - a markdown parser
```

Comment:
Good idea!

Reply:
Ah, I did forget the dark mode!! will do.

Reply:
I made a few simple tweaks to fit dark mode, mostly sticking to the default theme, maybe I'll add a full dark theme later.